Source/IGListSingleSectionController.m

```diff
@@ -53,6 +54,20 @@ - (instancetype)initWithNibName:(NSString *)nibName
     return self;
 }
 
+- (instancetype)initWithStoryboardCellIdentifier:(NSString *)identifier
+                              configureBlock:(IGListSingleSectionCellConfigureBlock)configureBlock
+                                   sizeBlock:(IGListSingleSectionCellSizeBlock)sizeBlock {
+    IGParameterAssert(identifier != nil);
+    IGParameterAssert(configureBlock != nil);
```

Comment:
@rnystrom What about the other two init funcions `initWithCellClass:` & `initWithNibName:` Do you want me to add  assertions for sizeBlock to them as well?